account/records.go

```diff
@@ -26,21 +27,29 @@ func CopyRecords(records Records) Records {
 
 // Record represents a single account record.
 type Record struct {
-	ID             uint32
-	Name           string
-	Email          string
+	ID              uint32
+	Name            string
+	Email           string
+	ClientViewHosts []string
+	DateCreated     string
+
+	// ClientViewURLS is used only by Version 2 clients. It is DEPRECATED
+	// and will go away when Version 2 is not longer supported.
+	// TODO remove when Version 2 is no longer supported.
 	ClientViewURLs []string
-	DateCreated    string
 }
 
 // CopyRecord deep copies a Record (it contains a pointer type).
 func CopyRecord(record Record) Record {
 	copy := Record{
-		ID:             record.ID,
-		Name:           record.Name,
-		Email:          record.Email,
-		ClientViewURLs: make([]string, 0, len(record.ClientViewURLs)),
-		DateCreated:    record.DateCreated,
+		ID:              record.ID,
+		Name:            record.Name,
+		Email:           record.Email,
+		ClientViewHosts: make([]string, 0, len(record.ClientViewHosts)),
```

Comment:
Maybe alloccate `ClientViewURLs` too since you are copying that below.

Reply:
i originally didn't because it is doomed, but went ahead and fixed in https://github.com/rocicorp/diff-server/pull/90